vars/testsPublishResults.groovy

```diff
@@ -156,6 +162,18 @@ def publishJMeterReport(Map settings = [:]){
     }
 }
 
+def publishCucumberReport(Map settings = [:]) {
+    if(settings.active){
```

Comment:
```suggestion
    if (settings.active) {
```

Reply:
Thanks, actually due to copy & paste, I'll address it in all occurrences in this step.